### ISSUE TITLE
Readme First example

### ISSUE THREAD

**Issue by nueverest** [twolfson/css-validator#4]
When I run the following I get `assert is not defined`.  I am running it [here](https://runkit.com/npm/css-validator).

```javascript
var validateCss = require('css-validator');
validateCss({text: 'a { color: blue; }'}, function (err, data) {
  assert.strictEqual(data.validity, true);
  assert.deepEqual(data.errors, []);
  assert.deepEqual(data.warnings, []);
});
```

**Comment by twolfson** [comment]
Thanks for the heads up, we will fix this by the end of next week. The corrected code below should work:

```javascript
var validateCss = require('css-validator');
var assert = require('assert');
validateCss({text: 'a { color: blue; }'}, function (err, data) {
  assert.strictEqual(data.validity, true);
  assert.deepEqual(data.errors, []);
  assert.deepEqual(data.warnings, []);
});
```

**Comment by twolfson** [comment]
This has been patched/released in `0.5.3`. Thanks for the report
